server/api/config.go

```diff
@@ -6,22 +6,31 @@ import (
 
 // StoredConfig represents the data stored in and managed with the Mattermost
 // config.
+//
+// StoredConfig should be abbreviated as sc.
 type StoredConfig struct {
-	Apps               map[string]interface{}
-	AWSAccessKeyID     string
-	AWSSecretAccessKey string
-}
+	// InstalledApps is a list of all apps installed on the Mattermost instance.
+	//
+	// For each installed app, an entry of string(AppID) -> sha1(App) is added,
+	// and the App struct is stored in KV under app_<sha1(App)>. Implemenrtation
+	// in `store.App`.
+	InstalledApps map[string]string `json:"installed_apps,omitempty"`
```

Comment:
If we are storing apps in the KV store why do we need this struct?

Reply:
To reload the KV values in memory upon changes. A change triggers a config change, triggers OnCOnfigurationChange on all servers, re-loading the cache